sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -154,7 +162,59 @@ func App() *cli.App {
 	return app
 }
 
-func runCtr(containerdSocket string, namespace string, containerID string, source string, superpowered bool) error {
+// Used to define valid container types
+type ContainerType string
+
+const (
+	Host      ContainerType = "host"
+	Bootstrap ContainerType = "bootstrap"
+)
+
+// IsValid returns true if an invalid type is valid
+func (ct ContainerType) IsValid() bool {
+	switch ct {
+	case Host, Bootstrap:
+		return true
+	}
+
+	return false
+}
+
+// PersistentDir returns the persistent base directory for the container type
+func (ct ContainerType) PersistentDir() (string, bool) {
+	switch ct {
+	case Host:
+		return "host-containers", true
+	case Bootstrap:
+		return "bootstrap-containers", true
+	}
+
+	return "", false
+}
```

Comment:
The call sites for these don't seem to use the third state, and is not idiomatic in this case (return the zero value). Can we drop this from the signature please?

```suggestion
// PersistentDir returns the persistent base directory for the container type
func (ct ContainerType) PersistentDir() string {
	switch ct {
	case Host:
		return "host-containers"
	case Bootstrap:
		return "bootstrap-containers"
	}

	return ""
}
```